styles.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: "myriadPro";
+  src: url(http://www.anaveer.in/honda/font/MYRIADPRO-REGULAR.OTF);
+ }
+ 
+body {
+  margin: 0;
+ }
+
+.main-container {
+  border-top: 5px solid #FFA500;
+  background-color: #FFF;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+ }
+
+.vertical-separator {
+  background: linear-gradient(to top, transparent, orange, transparent);
+  margin: 100px 40px;
+  width: 5px;
+  height: 250px;
+ }
+
+.feature-container {
+  background-color: #EEE;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding-top: 50px;
+  flex-wrap: wrap;
+  position: relative;
+ }
+
+.feature-container::before { 
+  content: " ";
+  background-color: #FFF;
+  background-repeat: repeat-x;
+  background-image: url("triangle.png");
+  height: 12px;
+  width: 100%;
+  top: 0;
+  position: absolute;
+  left: 0;
+ }
+ 
+.feature {
+  text-align: center;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+ }
+
+.feature-title {
+  font-size: 16xp;
+  font-family: "MyFont", Sans-Serif;
```

Comment:
font should be defined at a more general state because if not you'll have to set it for each element. just put it like this:
```
* {
  font-family: "myriadPro", Sans-Serif;
}
```
that says, set the font size specified to ALL the elements.